src/main/java/seedu/duke/commands/project/ProjectSelectCommand.java

```diff
@@ -25,25 +23,25 @@ public ProjectSelectCommand(HashMap<String, String> params)
     }
 
     public void parse() throws DukeExceptions {
-        try {
-            projectIndex = Integer.parseInt(getHashValue(params, "p")) - 1;
-            Parser.setProjectIndex(projectIndex);
-        } catch (NumberFormatException | IndexOutOfBoundsException e) {
-            throw new DukeExceptions("invalidProjectID");
-        }
+        projectIndex = Integer.parseInt(getHashValue(params, "p")) - 1;
+        Parser.setProjectIndex(projectIndex);
     }
 
-    public String executeCommand(ArrayList<Project> projects) throws DukeExceptions {
+    public String executeCommand(ArrayList<Project> projects, ArrayList<TeamMember> teamMembers) throws DukeExceptions {
         if (projects.size() == 0) {
             throw new DukeExceptions("emptyProjectList");
         }
-        Project selectedProject = projects.get(projectIndex);
-        System.out.println("Switched to Project \"" + selectedProject + "\"");
-        String projectDescription = selectedProject.getDescription();
-        String projectDeadline = selectedProject.getProjectDeadline();
-        //For small sam => help me implement this
-        String members = "<team members involved empty>";
-        return projectDescription + " | " + projectDeadline + " | " + members;
+        try {
+            Project selectedProject = projects.get(projectIndex);
+            System.out.println("Switched to Project \"" + selectedProject + "\"");
+            String projectDescription = selectedProject.getDescription();
+            String projectDeadline = selectedProject.getProjectDeadline();
+            //For small sam => help me implement this
```

Comment:
Just to clarify, this command activates the Project View both Samuels are doing right?

Reply:
They haven't implemented it yet so I'm not sure how to link it but yes that's the intention